docs/great-houses/house-of-ethereum.md

```diff
@@ -132,17 +132,15 @@ There are a bunch of wallets to choose from. **Your decision will mainly depend
 - Note: Whichever way you choose, you’ll likely need to store what we call a seed phrase. This is the most important bit. There are [many](https://www.google.com/search?client=firefox-b-d&q=storing+your+seed+phrase) [ways to do it](https://blog.trezor.io/https-blog-trezor-io-keep-your-seed-phrase-away-from-lions-edcc105457a0).
 - **NOTE: TAKE IT REALLY SERIOUSLY. STORE YOUR SEED PHRASE PHYSICALLY, NEVER COPY, PASTE OR ENTER IT ANYWHERE NEW
 
-## Layer 2 How-To
 
-Layer 2 is a collective term for solutions designed to help scale your application by handling transactions off the main Ethereum chain (layer 1). 
+## Layer 2 What
 
-- Understand [why layer 2 scaling is needed](https://ethereum.org/en/developers/docs/layer-2-scaling/) and learn about the different types of Layer 2 solutions.
-- Read more about the [State of Layer 2s](https://defiprime.com/ethereum-l2) as they are currently.
-- Instructions to [set up xDAI for MetaMask](https://xdaichain.com/for-users/wallets/metamask/metamask-setup) so that you can interact with the xDAI Layer.
-- How to [use the xDAI bridge](https://xdaichain.com/for-users/bridges/converting-xdai-via-bridge/moving-dai-to-xdai) to transfer assets from / to Ethereum mainnet.
-- Get a bit of xDAI to get you started at the [xDAI faucet](https://t.co/SXjJ0B7wRB?amp=1).
-- How to [configure Matic (Polygon) for MetaMask](https://docs.matic.network/docs/develop/metamask/config-matic/) to interact with dApps on Matic (Polygon).
+Layer 2 is a collective term for solutions designed to help scale your application by handling transactions off the main Ethereum chain (layer 1). 
 
+- What is [layer 2?](https://ethereum.org/en/layer-2/)  
+- Check out the [House of L2s and Sidechains](https://wiki.metagame.wtf/great-houses/house-of-L2s)
+- Follow this Playbook to [configure Polygon for MetaMask and get some gas](https://wiki.metagame.wtf/playbooks/how-to-install-wallet-get-gas.md) 
```

Comment:
This link takes the user to a Page not found error page. When you're linking to pages in the wiki, you don't want to include the file extension `.md`.

Reply:
@tommycomma you'll also want to make sure that the `.md` extension is removed from the MM playbook URL